packages/lib-classifier/src/plugins/tasks/drawing/models/DrawingTask.js

```diff
@@ -98,6 +98,8 @@ export const Drawing = types.model('Drawing', {
         self.tools.forEach(tool => tool.reset())
         self.activeToolIndex = 0
         self.subTaskVisibility = false
+        self.hidingIndex = null
+        self.shownMarks = SHOWN_MARKS.ALL
```

Comment:
This line will need to be modified slightly for transcription tasks, where `shownMarks` has three possible states, not two.